Cognito migration guide (#1115)
FusionAuth/fusionauth-site: site/docs/v1/tech/migration-guide/_what-next-cognito.adoc

> If you want to migrate Client Credentials grants, create Entities in FusionAuth corresponding to each Cognito app client that is configured to use the Client Credentials grant. The scopes within FusionAuth are currently constrained to a certain format. link:/docs/v1/tech/core-concepts/entity-management/[Learn more about Entity Management].

Reviewer: ```suggestion
If you want to migrate Client Credentials grants, create Entities in FusionAuth corresponding to each Cognito app client that is configured to use the Client Credentials grant. The scopes within FusionAuth are currently constrained to a certain format. link:/docs/v1/tech/core-concepts/entity-management/[Learn more about Entity Management].
```

This reads awkwardly

Reviewer: ```suggestion
If you want to migrate Client Credentials from Cognito, create the corresponding Entities in FusionAuth to each Cognito app client (currently using the grant). Keep in mind, scopes within FusionAuth are currently constrained to a certain format, which may inform how you migrate. link:/docs/v1/tech/core-concepts/entity-management/[Learn more about Entity Management].
```

> You can migrate each of your Cognito app clients to a FusionAuth application using the link:/docs/v1/tech/apis/applications/#create-an-application[Application API], which allows you to set your Client Id and Client Secret to be the same as they were in Cognit. Maintaining these values will minimize impact on any applications using the OAuth Authorization Code grant or Implicit grant. Ensure you add the application to the correct tenant.

Reviewer: ```suggestion
You can migrate each of your Cognito app clients to a FusionAuth application using the link:/docs/v1/tech/apis/applications/#create-an-application[Application API], which allows you to set your Client Id and Client Secret to be the same as they were in Cognito. Maintaining these values will minimize the impact on applications using the OAuth Authorization Code grant or Implicit grant (read: a more seamless experience). Ensure you add the application to the correct tenant.
```
